bin/docker_utils.py

```diff
@@ -23,7 +23,13 @@ def hub_request(uri, data=None, params=None, headers=None, method='GET', json=Fa
 
 def http_request(url, data=None, params=None, headers=None, method = 'GET', json=False):
   response = request(method=method, url=url, data=data,  params=params, headers=headers)
-  return response.json() if json else response
+  if json:
+    try:
+      response = response.json()
+    except ValueError as e:
+      print("No json response returned: ", e)
+      response = response.text
+  return response
```

Comment:
I think this change is not needed.  Just getting a non-json response for `logout` should be enough .... right?